RELEASE/scripts/autoscend/auto_consume.ash

```diff
@@ -1032,28 +1032,28 @@ boolean loadConsumables(string _type, ConsumeAction[int] actions)
 		{
 			int n = count(actions);
 			actions[n] = MakeConsumeAction(it);
-			if (obtain_mode == SL_OBTAIN_PULL)
+			if (obtain_mode == AUTO_OBTAIN_PULL)
 			{
 				actions[n].desirability -= 5.0;
 				float user_desirability = get_property("auto_consumePullDesirability").to_float();
 				if (user_desirability > 0.0)
 				{
-					actions[n].desirability -= user_desirability;
+					actions[n].desirability = -user_desirability;
```

Comment:
this actually works? I thought you would need to do
`actions[n].desirability = actions[n].desirability-user_desirability;`